browser/app/js/browser/Header.js

```diff
@@ -16,28 +16,64 @@
 
 import React from "react"
 import Path from "../objects/Path"
-import StorageInfo from "./StorageInfo"
-import BrowserDropdown from "./BrowserDropdown"
 import web from "../web"
 import { minioBrowserPrefix } from "../constants"
+import { connect } from "react-redux"
+import * as actionsCommon from "./actions"
+import history from "../history"
+import ReactTooltip from 'react-tooltip'
+
+export class Header extends React.Component {
+  logout(e) {
+    e.preventDefault()
+    web.Logout()
+    history.replace("/login")
+  }
+  render() {
+    const loggedIn = web.LoggedIn()
+    const { toggleSidebar } = this.props
+
+    return (
+      <header className="header">
+        <div className="sidebar-toggle" onClick={toggleSidebar} />
+
+        <Path />
 
-export const Header = () => {
-  const loggedIn = web.LoggedIn()
-  return (
-    <header className="fe-header">
-      <Path />
-      {loggedIn && <StorageInfo />}
-      <ul className="feh-actions">
         {loggedIn ? (
-          <BrowserDropdown />
+          <React.Fragment>
+            <a
+              data-tip="Sign Out"
+              data-for="tooltip-sign-out"
+              onClick={this.logout}
+              className="sign-out"
+            />
+            <ReactTooltip
+              id="tooltip-sign-out"
+              effect="solid"
+              place="left"
+              className="tooltip"
+            />
+          </React.Fragment>
         ) : (
-          <a className="btn btn-danger" href={minioBrowserPrefix + "/login"}>
+          <a className="guest-login" href={minioBrowserPrefix + "/login"}>
             Login
           </a>
         )}
-      </ul>
-    </header>
-  )
+      </header>
+    )
+  }
+}
+
+const mapStateToProps = state => {
```

Comment:
Similar to my previous command looks like we aren't using sidebarOpen anywhere in this file so we can remove the mapStateToProps here and change line 79 to `export default connect(undefined, mapDispatchToProps)(Header)`